packages/docs/src/page-configs/ui-elements/scroll-container/examples/Color.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <va-scroll-container vertical color="primary" style="max-height: 300px;">
+    <va-list>
+      <va-list-item v-for="i in 30" :key="i">
+        {{ i}}
```

Comment:
We definitely need to enable linter for the docs examples too.

Reply:
I just committed without linter cuz there was a lot of lint changes in docs, see: https://github.com/epicmaxco/vuestic-ui/pull/2321